src/components/view/sources/sources.ts

```diff
@@ -0,0 +1,24 @@
+import './sources.css';
+
+class Sources {
+  draw(data: [{ name: string; id: string }]): void {
+    const fragment = document.createDocumentFragment();
+    const sourceItemTemplate = document.querySelector('#sourceItemTemp') as HTMLTemplateElement;
+    if (sourceItemTemplate) {
+      data.forEach((item) => {
+        const sourceClone = sourceItemTemplate.content.cloneNode(true) as HTMLElement;
+        if (sourceClone) {
+          const itemNameElement = sourceClone.querySelector('.source__item-name') as HTMLElement;
+          if (itemNameElement) itemNameElement.textContent = item.name;
```

Comment:
weird, prettier usually formats lines like this and wraps them with curly braces. Did you setup it right?